docs/articles/playbooks/Expensify-Playbook-for-Small-to-Medium-Sized-Businesses.md

```diff
@@ -100,7 +100,7 @@ This is essentially like setting a daily or individual expense limitation on any
 *Receipt Required Amount: $75*
 Receipts are important, and in most cases you prefer an itemized receipt. However, Expensify will generate an IRS-compliant electronic receipt (not itemized) for every expense not tied to hotels expense. For this reason, it’s important to enforce a rule where anytime an employee is on the road, and making business-related purchases at hotel (which happens a lot!), they are required to attach a physical receipt. 
 
-![Expense Basics](https://help.expensify.com/assets/images/playbook-expense-basics.png)
+![Expense Basics](https://help.expensify.com/assets/images/playbook-expense-basics.png){:width="100%"}
```

Comment:
Both browser and vscode preview parse this incorrectly somehow
<img width="1169" alt="Screenshot 2023-08-07 at 20 58 26" src="https://github.com/Expensify/App/assets/9059945/45c1aea5-6756-446a-b87b-757c63a23059">

Reply:
hm, just this one? Or all of them? If all of them, I think it may be a limitation of the markdown - vs. the actual jekyll server. Were you trying the server or just the markdown?

Reply:
> Or all of them?

It's all of them

> Were you trying the server or just the markdown?

I'm just looking at how it looks in GitHub preview ([example](https://github.com/Expensify/App/blob/189bc74c9dc87b188581f73b38c361b598d22fea/docs/articles/playbooks/Expensify-Playbook-for-Small-to-Medium-Sized-Businesses.md))